docs/app/pages/home/material-landing/material-landing.component.ts

```diff
@@ -10,7 +11,10 @@ import { NbThemeService } from '@nebular/theme';
   ],
 })
 export class MaterialLandingComponent {
-  constructor(private themeService: NbThemeService) {
+  constructor(private themeService: NbThemeService, private metadataService: MetadataService) {
+    this.metadataService.updateDescription('Ngx-admin material works perfectly with Angular Material and Nebular.' +
+      ' Over 40+ Angular Components and 60+ Usage Examples.Take the best from both!');
+    this.metadataService.updateTitle('New Material theme of ngx-admin dashboard template based on Angular 9+');
```

Comment:
Title should be
`Ngx-admin material dashboard template based on Angular 9+ and Bootstrap 4+`

Reply:
fix